src/analysis/normed_space/basic.lean

```diff
@@ -86,6 +87,10 @@ noncomputable def semi_normed_group.of_core (α : Type*) [add_comm_group α] [ha
     calc ∥x - y∥ = ∥ -(y - x)∥ : by simp
              ... = ∥y - x∥ : by { rw [C.norm_neg] } }
 
+instance : normed_group punit :=
```

Comment:
In fact, maybe `punit` is even a `normed_comm_ring`? Or do the axioms require `zero_lt_one`? I forgot.

Reply:
Done.